frontend/src/Components/ProductsFilterSection.js

```diff
@@ -0,0 +1,120 @@
+import { useRef } from "react";
+import { useProduct } from "../Contexts/productContext";
+import RangeSlider from "./RangeSlider";
+
+const ProductsFilterSection = () => {
+  const sortRef = useRef();
+  const thumbRef = useRef();
+  const { priceRange, dispatch, categoriesList, sortByRating, productsList } =
+    useProduct();
+
+  const handlePriceRangeChange = (range) => {
+    dispatch({
+      type: "PRICE_RANGE",
+      payload: range,
+    });
+  };
+
+  const handleCategoryChange = (item, e) => {
+    dispatch({
+      type: "CATEGORY",
+      payload: { [item]: e.target.checked },
+    });
+  };
+
+  const handleClearFilters = (e) => {
+    sortRef.current.value = "DEFAULT";
+    dispatch({
+      type: "CLEAR_FILTER",
+      payload: productsList,
+    });
+  };
+
+  return (
+    <div className="product-page__filters">
+      <div className="heading">
+        <h2>Filters</h2>
+        <button
+          className="btn btn--sm btn--outline"
+          onClick={(e) => handleClearFilters(e)}
```

Comment:
I think, there is no need to pass `e` here like this. As javascript will send `e` automatically. So you can simply do this 
```javascript
onClick={handleClearFilters}
```

And also, you are not using `e` in the function, there is no need to pass and accept it as a param.